Detectors/EMCAL/simulation/include/EMCALSimulation/LZEROElectronics.h

```diff
@@ -76,6 +78,10 @@ class LZEROElectronics
   /// Sets geometry for trigger mapping
   void setGeometry(o2::emcal::Geometry* gm) { mGeometry = gm; }
 
+  /// Sets the masked fastOrs from the CCDB in the LZERO
+  void setMaskedFastOrs(std::vector<int> maskedfastors) { mMaskedFastOrs = maskedfastors; }
```

Comment:
pass `maskedfastors` as `const std::vector<int>& maskedfastors`

Reply:
Hi @shahor02 , does it mean that once I implement this change you are able to approve this PR and merge it, so you have the missing rights @jokonig and I need?